include/osquery/tables.h

```diff
@@ -67,7 +67,7 @@ namespace tables {
 
 /// Helper alias for TablePlugin names.
 typedef const std::string TableName;
-typedef const std::vector<std::pair<std::string, std::string> > TableColumns;
+typedef std::vector<std::pair<std::string, std::string> > TableColumns;
```

Comment:
I needed to remove the `const` on this to use it in the `getQueryColumns` function. I added a `const` in the only place I could find the type used in non-generated code.